doc/github_handler.rst

```diff
@@ -58,23 +58,36 @@ On this picture numbers depict the following events:
 
 1. User makes a new commit
 2. GitHub detects a commit, creates a check suite and sends a web-hook payload to Jenkins server
-3. A preconfigured Jenkins job passes received parameters to GitHub Handler
-4. GitHub Handler checks that repo name and web-hook event are applicable and and sends a request
+3. `Generic Webhook Trigger` plugin triggers a preconfigured Jenkins job ('GitHub Webhook handler') on an already set up
+   automation server, passing required parameters, such as payload contents and "x-github-event" header
+4. GitHub Handler checks that repo name and web-hook event are applicable, and if so, sends a request
```

Comment:
I think you can mention that GitHub Handler uses GitHub API for creating check runs.